CHANGES.md

```diff
@@ -24,6 +24,19 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- new features
+  - added support for causal, counterfactual, and error analysis tools in responsibleai
+  - added model analysis widget to raiwidgets
```

Comment:
missing: add regression support to error analysis

Reply:
Added